packages/react/src/components/Forms/DatePicker/index.scss

```diff
@@ -0,0 +1,255 @@
+      border-color: var(--c--theme--colors--greyscale-300);
+      cursor: default;
+      pointer-events: none;
+
+      label {
+        color: var(--c--theme--colors--greyscale-400);
+      }
+
+      &__toggle {
+        color: var(--c--theme--colors--greyscale-400);
+      }
+
+      .c__date-picker__inner {
+        color: var(--c--theme--colors--greyscale-600);
+        &__action {
+          color: var(--c--theme--colors--greyscale-400);
+        }
+      }
+
+      label {
+        cursor: default;
+      }
+    }
+  }
+
+  &--invalid {
+    .c__date-picker__wrapper {
+      border-color: var(--c--theme--colors--danger-600);
+      border-radius: var(--c--components--forms-datepicker--border-radius);
+
+      &__toggle {
+        color: var(--c--theme--colors--danger-600);
+      }
+
+      label {
+        color: var(--c--theme--colors--danger-600);
+      }
+    }
+
+    &:not(.c__date-picker--disabled) {
+      &:hover {
+        .c__date-picker__wrapper {
+          border-color: var(--c--theme--colors--danger-200);
+          &__toggle {
+            color: var(--c--theme--colors--danger-200);
+          }
+        }
+      }
+    }
+  }
+
+  &--picked {
+    .c__date-picker__wrapper {
+      border-radius: var(--c--components--forms-datepicker--border-radius--focus);
+      border-color: var(--c--components--forms-datepicker--border-color--focus);
+
+      &__toggle {
+        color: var(--c--components--forms-datepicker--border-color--focus);
+      }
+    }}
+
+  &:not(&--picked):not(&--invalid):not(&--disabled) {
+    &:hover {
+      .c__date-picker__wrapper {
+        border-radius: var(--c--components--forms-datepicker--border-radius--hover);
+        border-color: var(--c--components--forms-datepicker--border-color--hover);
+      }
+    }
+  }
+
+}
+
+
+.c__calendar {
+  display: block;
+  transform: translate(2px, 0);
+
+  &__wrapper {
+    display: block;
+    padding: 0.75em;
+
+    &--opened {
+      box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.3);
+      background-color: var(--c--components--forms-datepicker--menu-background-color);
+    }
+
+    &__header {
+      display: flex;
+      justify-content: space-between;
+
+      &__actions {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+
+        overflow: hidden;
+        white-space: nowrap;
+        min-width: 180px;
+      }
+    }
+
+    &__grid {
+      width: 100%;
+      padding-top: 0.5em;
+
+      &__header-row {
+        line-height: 4em;
+        font-size: var(--c--components--button--small-font-size);
+
+        th {
+          font-weight: var(--c--components--button--font-weight);
+          color: var(--c--theme--colors--greyscale-700);
+        }
+      }
+
+      &__week-row {
+        &__button {
+          margin: 0.3em auto;
+          &--selected {
+            border-radius: 100%;
+          }
+        }
+        .c__button--small {
+          height: 27.63px;
```

Comment:
We try to use rem as much as possible + why do you need to do that? (Might be related to my last comment about the fact that we should try to remove non clickabe areas)

Reply:
I see. I have chosen `px` because the `--c--components--button--small-height` tokens was in px.

Reply:
I force the height and the width of the button to make sure it's a square one.

Reply:
Gonna refactor these values to em !